taxi/models.py

```diff
@@ -0,0 +1,45 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+
+from django.db import models
+from django.db.models import CASCADE
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(
+        max_length=63,
+        unique=True,
+    )
+
+    class Meta:
+        ordering = ["license_number", ]
```

Comment:
According to the code style guide, the `Meta` class should be defined after all the attributes of the model. Please move the `Meta` class to the end of the `Driver` model.